shapeshift/web
# feat: route to ETH/FOX staking without AccountId

## Description



Makes ETH/FOX staking routes AccountId-agnostic so that #4073 can consume it and route to rewards step




## Pull Request Type


## Issue (if applicable)



N/A

## Risk



Low/none, though we should sanity check the current claim logic still
claim from the selected account

## Testing



- Click "Claim" from a given AccountId ETH/FOX staking overview modal
- Ensure the claim step reflects the previously selected account

### Engineering



- ^

### Operations



- ^

## Screenshots (if applicable)

<img width="1594" alt="image" src="https://user-images.githubusercontent.com/17035424/226388112-933864c2-f7e8-41fb-b8bd-d7628128e6fc.png">
